nvmbugs/001_simple/intrinsics.h

```diff
@@ -0,0 +1,10 @@
+#ifndef INTRINSICS_H
```

Comment:
Can you add a rule so anything under the `nvmbugs/` can easily include this? Low priority